src/Bloc-Layout/BlProportionalLayout.class.st

```diff
@@ -51,17 +55,26 @@ BlProportionalLayout class >> constraints [
 { #category : #measure }
 BlProportionalLayout >> boundsForChild: aChild in: parentBounds [
 
-	| result horizontalProportions verticalProportions |
+	| horizontalProportions verticalProportions result |
 	horizontalProportions := aChild constraints proportional horizontal.
 	verticalProportions := aChild constraints proportional vertical.
 
-	result := BlBounds fromRectangle: (aChild margin inset: parentBounds).
+	result := BlBounds fromRectangle: parentBounds.
```

Comment:
Here is where you can see that margin was applied first, and the proportions are applied secondly.

With this change, now first you apply the proportions, and second the margin. I think it's correct.